docs/custom.css

```
@@ -31,7 +31,7 @@
 
 @import 'tailwindcss' source('./src/');
 @import 'tw-animate-css';
-
+@import '@mastra/docusaurus-plugin-kapa';
```

Stylelint (17.14.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)